joomla/joomla-cms
# [6.0] Replace dialog GIF loader with Custom Element

### Summary of Changes

This PR replaces the GIF loading icon used within dialogs, with the Custom Element.

### Testing Instructions

Note: This cannot be tested with Patch Tester!

1. Go to edit any article.
2. Click "Versions" in the toolbar.
3. Before the list of versions are loaded, you should see a loading icon for a split second.

### Actual result BEFORE applying this Pull Request
![image](https://github.com/user-attachments/assets/fd4ffa80-2a85-4143-854a-b5cbbc73373c)

### Expected result AFTER applying this Pull Request
![image](https://github.com/user-attachments/assets/f55ae5b1-b359-42e7-9e64-db67d15809e1)

cc'ing @Fedik @dgrammatiko
